api/swagger-spec/oapi-v1.json

```diff
@@ -13819,6 +13819,11 @@
      "resources": {
       "$ref": "v1.ResourceRequirements",
       "description": "the desired compute resources the build should have"
+     },
+     "maxDuration": {
```

Comment:
@bparees your choice,  personally I'd go with `buildDeadlineSeconds`
On Oct 6, 2015 11:12 PM, "Clayton Coleman" <notifications@github.com> wrote:

> In api/swagger-spec/oapi-v1.json
> <https://github.com/openshift/origin/pull/4821#discussion_r41323078>:
>
> > @@ -13819,6 +13819,11 @@
> >       "resources": {
> >        "$ref": "v1.ResourceRequirements",
> >        "description": "the desired compute resources the build should have"
> > +     },
> > +     "maxDuration": {
>
> Anything you name this has to have a suffix of "DeadlineSeconds" because
> that's the established convention in the API. You can call it
> buildDeadlineSeconds or activeDeadlineSeconds or (other variant), but it
> has to fit that pattern. On Oct 6, 2015, at 4:50 PM, Maciej Szulik <
> notifications@github.com> wrote: In api/swagger-spec/oapi-v1.json <
> https://github.com/openshift/origin/pull/4821#discussion_r41320288>:
> @@ -13819,6 +13819,11 @@ "resources": { "$ref": "v1.ResourceRequirements",
> "description": "the desired compute resources the build should have" + }, +
> "maxDuration": {
> We've talked with @bparees <https://github.com/bparees> <
> https://github.com/bparees> about naming and decided to go with
> maxDuration as it speaks more clearly what the field does than
> activeDeadlineSecods. Other than that if sometime in the future we'll
> decide to allow multi-pod builds, we'd still have maxDuration which will
> leverage job's maxDuration (or whatever it'll be called). — Reply to this
> email directly or view it on GitHub <
> https://github.com/openshift/origin/pull/4821/files#r41320288>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4821/files#r41323078>.
>